__tests__/unittest/extendOrdWithCustom.test.js

```diff
@@ -1,116 +1,151 @@
+        it("should skip if customOrdContentFile property in the .cdsrc.json points to NON-EXISTING custom ord file", () => {
             const ordContent = {};
             appConfig.env.customOrdContentFile = "./ord/NotExistingCustom.ord.json";
-            const result = extendCustomORDContentIfExists(appConfig, ordContent);
+            const result = extendCustomORDContentIfExists(
+                appConfig,
+                ordContent
+            );
             expect(result).toEqual(ordContent);
         });
 
-        it('should ignore and log warn if found ord top-level primitive property in customOrdFile', () => {
+        it("should ignore and log warn if found ord top-level primitive property in customOrdFile", () => {
             const ordContent = {};
-            const warningSpy = jest.spyOn(console, 'warn');
-            prepareTestEnvironment({ namespace: "sap.sample" }, appConfig, 'testCustomORDContentFileThrowErrors.json');
-            const result = extendCustomORDContentIfExists(appConfig, ordContent);
+            prepareTestEnvironment({ namespace: "sap.sample" }, appConfig, "testCustomORDContentFileThrowErrors.json");
+            const result = extendCustomORDContentIfExists(appConfig,ordContent);
 
             expect(warningSpy).toHaveBeenCalledTimes(3);
-            expect(warningSpy).toHaveBeenCalledWith('Mocked warning');
-
+            expect(warningSpy).toHaveBeenCalledWith(
+                "[ord-plugin] -",
+                expect.stringContaining("Found ord top level primitive ord property in customOrdFile:"),
+                expect.anything(),
+                expect.stringContaining("Please define it in .cdsrc.json."));
             expect(result).toMatchSnapshot();
         });
 
-        it('should add new ord resources that are not supported by cap framework', () => {
+        it("should add new ord resources that are not supported by cap framework", () => {
             const ordContent = {};
-            prepareTestEnvironment({}, appConfig, 'testCustomORDContentFileWithNewResources.json');
-            const result = extendCustomORDContentIfExists(appConfig, ordContent);
+            prepareTestEnvironment(
+                {},
+                appConfig,
+                "testCustomORDContentFileWithNewResources.json"
+            );
+            const result = extendCustomORDContentIfExists(
+                appConfig,
+                ordContent
+            );
             expect(result).toMatchSnapshot();
         });
 
-        it('should enhance the list of generated ord resources', () => {
-            const ordContent = { packages: [{ ordId: "sap.sm:package:smDataProducts:v1", localId: "smDataProductsV1" }] };
-            prepareTestEnvironment({}, appConfig, 'testCustomORDContentFileWithEnhanced.json');
-            const result = extendCustomORDContentIfExists(appConfig, ordContent);
+        it("should enhance the list of generated ord resources", () => {
```

Comment:
I would suggest that set this format in .vscode settings, and then we don't need to take care of it in the future

Reply:
You mean - `formatter`, right? Sure, we can do that, but I don't think it's urgent to be done in this PR.

Reply:
Following: https://github.com/cap-js/ord/issues/100